### PR TITLE
Regression test looks for extension crash described in the related issue #1698

### PR DESCRIPTION
**Pull request by timstackblock**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1279083466).

I made all the revision mentioned for PR 1719 here.

This is a regression test to look for web wallet crashes that are caused by entering more than 7 decimal places. The test attempts to reproduce the P1 bug we found in issue #1698 as well as additional test that test edge cases related to that functionality.

@UXteam
cc/ @aulneau @kyranjamie @fbwoolf @beguene @markmhx @Eshwari007 @andresgalante

### CODE REVIEW

File: src/pages/transaction-signing/components/event-card.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ export const TransactionEventCard: React.FC<any> = ({
       <Stack spacing="base-loose" p="base-loose">
         <SpaceBetween position="relative">
           <Text fontWeight={500} fontSize={2}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
```
Comment (kyranjamie) [inline src/pages/transaction-signing/components/event-card.tsx:25]: Apply id to `<Text>` element, a new span isn't necessary

File: README.md [hunk 2]
```diff
@@ -163,3 +163,4 @@ Please note this email is strictly for reporting security vulnerabilities. For s
 In Q1 2021, Hiro partnered with [Least Authority](https://leastauthority.com/), a leading security consultancy with experience in the crypto space, to audit Hiro Wallet for Web. On April 29th 2021, after addressing the major concerns described in the initial findings, as well as a concluding sign off from the Least Authority team, a final report was delivered.
 
 [Download and read the full report here](https://github.com/blockstack/stacks-wallet-web/blob/main/public/docs/least-authority-security-audit-report.pdf)
+timtest123456
```
Comment (kyranjamie) [inline README.md]: To remove

**Review by kyranjamie — COMMENTED** [review]
Please rebase your commits into a single 1 using git rebase https://dev.to/blakedeboer/beginners-guide-to-interactive-rebasing-1ob

`git rebase -i origin/master`

and change all but the first to `fixup`

File: README.md (hunk 2 above)
Comment (timstackblock) [reply README.md]: Done

File: src/pages/transaction-signing/components/event-card.tsx (hunk 1 above)
Comment (timstackblock) [reply src/pages/transaction-signing/components/event-card.tsx:25]: I have problems grabbing the element with just the <Text> I do this to avoid flakes. It works but the test is brittle and breaks for me on some runs.

Comment (kyranjamie) [reply src/pages/transaction-signing/components/event-card.tsx:25]: I have a feel that's nothing to do with the selector being on the text element (which underneath is a span). It's either there or it isn't.

Comment (timstackblock) [reply src/pages/transaction-signing/components/event-card.tsx:25]: this rebase is not working for me

Comment (kyranjamie) [reply src/pages/transaction-signing/components/event-card.tsx:25]: what's going wrong?

Comment (timstackblock) [reply src/pages/transaction-signing/components/event-card.tsx:25]: Let jump on a zoom when you have a free second the rebase isnt on a branch

 (no branch, rebasing chore/regressionSep27)

Comment (timstackblock) [reply src/pages/transaction-signing/components/event-card.tsx:25]: resolved

**Comment by timstackblock** [comment]
@kyranjamie please approve so I can merge

**Review by kyranjamie — APPROVED** [review]
